modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/defender.dm

```diff
@@ -0,0 +1,115 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+/mob/living/carbon/alien/humanoid/skyrat/defender
+	name = "alien defender"
+	caste = "defender"
+	maxHealth = 300
+	health = 300
+	icon_state = "aliendefender"
+	melee_damage_lower = 25
+	melee_damage_upper = 30
+	/// Holds the basic charge ability that the defender will be granted
+	var/datum/action/cooldown/mob_cooldown/charge/basic_charge/defender/charge
+	/// Holds the wrecking ball tail sweep that the defender will be granted
+	var/datum/action/cooldown/spell/aoe/repulse/xeno/crushing/tail_sweep
+	next_evolution = /mob/living/carbon/alien/humanoid/skyrat/warrior
+
+/mob/living/carbon/alien/humanoid/skyrat/defender/Initialize(mapload)
+	. = ..()
+	tail_sweep = new /datum/action/cooldown/spell/aoe/repulse/xeno/crushing()
+	tail_sweep.Grant(src)
+
+	charge = new /datum/action/cooldown/mob_cooldown/charge/basic_charge/defender()
+	charge.Grant(src)
```

Comment:
reference 1.2 (abilities in the ability list would be initialized in a parent instead)